tests/translators/test_translators.py

```diff
@@ -133,7 +134,7 @@ def test_gpd_dataframe_translator():
 
 def test_transform_from_grib_file():
     # transform grib-based data object
-    f = from_source("file", "tests/data/test_single.grib")
+    f = from_source("file", earthkit_test_data_file("test_single.grib"))
```

Comment:
Hi @sandorkertesz ,
I had to include this to get the tests to pass. It is probably the correct way to load files, but I am quite confused why this only fails on my PR as my code change should not touch these tests. Any ideas? Or are you happy for this improvement to the tests to be included in this PR.
Eddy

Reply:
Yes,  `earthkit_test_data_file()` is right method to use here.